src/components/_pages/PageStrategy.tsx

```diff
@@ -14,7 +14,7 @@ export const PageStrategy: NextPage<StrategyLandingPageProps> = ({
   sectionStrategies,
 }) => {
   return (
-    <Layout>
+    <Layout px={4}>
```

Comment:
This may cause issues with double padding where the `<Container />` component is used. I don't think this needs to be reversed, just something to keep an eye out for if you notice awkward spacing at different viewport widths